kolibri/core/tasks/test/test_decorators.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from kolibri.core.tasks.decorators import task
+from kolibri.core.tasks.exceptions import FunctionNotRegisteredAsJob
+from kolibri.core.tasks.job import JobRegistry
+from kolibri.core.tasks.job import RegisteredJob
+from kolibri.core.tasks.utils import stringify_func
+
+
+@pytest.fixture
+def registered_jobs():
+    JobRegistry.REGISTERED_JOBS.clear()
+    yield JobRegistry.REGISTERED_JOBS
+    JobRegistry.REGISTERED_JOBS.clear()
+
+
+class TestTaskDecorators(object):
```

Comment:
Could you make this a [django `TestCase` ](https://docs.djangoproject.com/en/3.2/topics/testing/overview/)and use the assertion methods they provided? (e.g. `self.assertEqual`)? This is just to be consistent with the rest of the test suite; plus you might need/want some of the features provided to the test runner.

Reply:
Yeap, good idea. I've pushed commits that addresses this.